docs/content/preview/yugabyte-voyager/known-issues/postgresql.md

```diff
@@ -1615,3 +1605,190 @@ Suggested change to the schema is to remove this redundant index `idx_orders_ord
+
+**Description**:
+
+In YugabyteDB, the columns specified in the [CREATE INDEX](../../../api/ysql/the-sql-language/statements/ddl_create_index) statement are of three kinds: sharding, clustering, and covering. For more details, refer to the documentation for [secondary indexes](../../../explore/ysql-language-features/indexes-constraints/secondary-indexes-ysql/). The sharding strategy is HASH by default unless [Postgres Compatibility mode](./../../develop/postgresql-compatibility/) is enabled, in which case, RANGE is the default sharding strategy.
+
+Design the index to evenly distribute data across all nodes and optimize performance based on query patterns.
+
+If the index is designed for a column with a high percentage of a particular value in the data, it can create a hotspot issue. All the data for that value will reside on a single tablet, which will become a hotspot for this value in the queries and, hence, cause performance degradation.
+
+**Workaround**: If the frequently occurring value is not being queried, it is recommended that a Partial index be created by filtering this value and, hence, optimizing it for other data.
+
+If the value is being queried and the index is a single-column index, it is recommended to add another column and make it a multi-column range-sharded index to distribute the value evenly across various nodes. If the index is multi-column, it is recommended to make it a range-sharded index.
+
+**Example**
+
+An example schema on the source database is as follows:
+
+```sql
+CREATE TABLE user_activity (
+    user_id int PRIMARY,
+    event_type text, --type of the activity 'login', 'logout', 'profile_update
+, 'email_verification', so on.. various events 
+    event_timestamp timestampz,
+    ...
+);
+
+CREATE INDEX idx_user_activity_event_type on user_activity (event_type); --this index is on the event_type which is having 80% data with 'login' type 
+
+CREATE INDEX idx_user_activity_event_type_user_id on user_activity (event_type, user_id); --this index is on the event_type which is having 80% data with 'login' type 
+
+```
+
+As these indexes have a sharding key on the event_type column, which has the most common value as ‘login’, which covers 80% of the data, and hence 80% of the data resides on a single tablet, which becomes a hotspot.
+
+Suggested change to the schema is to
+
+Partial indexing by removing the ‘login’ value from the index to optimize it for other values.
+
+```sql
+CREATE INDEX idx_user_activity_event_type on user_activity (event_type) where event_type <> 'login' ; --filtering the 'login' values so those will not be indexed
+
+CREATE INDEX idx_user_activity_event_type_user_id on user_activity (event_type, user_id) where event_type <> 'login' ;  --filtering the 'login' values so those will not be indexed
+```
+
+OR
+
+Explicitly making it a range-sharded index so that the empty string value is evenly distributed across all nodes with another column.
```

Comment:
```suggestion
Explicitly making it a range-sharded index so that the empty string value is evenly distributed across all nodes by adding another column.
```